Extensions/TopDownMovementBehavior/JsExtension.cpp

```diff
@@ -74,6 +74,11 @@ class TopDownMovementBehaviorJsExtension : public gd::PlatformExtension {
         "getXVelocity");
     autConditions["TopDownMovementBehavior::YVelocity"].SetFunctionName(
         "getYVelocity");
+    autActions["TopDownMovementBehavior::MovementAngleOffset"]
+        .SetFunctionName("setMovementAngleOffset")
+        .SetGetter("getMovementAngleOffset");
+    autConditions["TopDownMovementBehavior::MovementAngleOffset"].SetFunctionName(
+        "getMovementAngleOffset");
```

Comment:
Did you intend to do one action/condition for it? I don't see a declaration for it.
I recommend also adding an expression :)

You can do all of them in one since recently thanks to `AddExpressionAndConditionAndAction`:
```
behavior.AddExpressionAndConditionAndAction("number", "MovementAngleOffset", _("Movement angle offset"), _("the movement angle offset"), _("the movement angle offset"), _("Movement"), "CppPlatform/Extensions/topdownmovementicon24.png")
      .AddParameter("object", _("Object"))
      .AddParameter("behavior", _("Behavior"), "TopDownMovementBehavior")
      .UseStandardParameters("number")
```

Note that this will add a "MovementAngleOffset" expression (as expected), a condition called "TopDownMovementBehavior::MovementAngleOffset" (as you did), and an action called ""TopDownMovementBehavior::SetMovementAngleOffset" (there is a "Set" that is added, which should be the norm for actions).

If you don't get this to work, let me know and I can do it myself.

Reply:
Yes, I couldn't make it work. It doesn't compile and I didn't find any example in other extensions.

Reply:
There was a missing include file, I've fixed this so it works for all extensions in the future.